resources/prosody-plugins/mod_presence_identity.lua

```diff
@@ -0,0 +1,40 @@
+local stanza = require "util.stanza";
+local core_post_stanza = prosody.core_post_stanza;
+
+-- For all received presence messages, if the jitsi_meet_context_(user|group)
+-- values are set in the session, then insert them into the presence messages
+-- for that session.
+function on_message(event)
+    if event["stanza"] then
+      local presence = event["stanza"]
```

Comment:
Do you use this local 'presence' variable anywhere here?  Seems like you use new_stanza from the stanza.clone(event.stanza), skipping 'presence' other than here.

Reply:
yup you're right.  i'll either remove this or change the other code to use it.